core/controllers/base.py

```diff
@@ -425,6 +461,7 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
```

Comment:
Thanks for the guidance. Found the answer in the [documentation](https://webapp2.readthedocs.io/en/latest/api/webapp2.html#webapp2.WSGIApplication.handle_exception). Pushed the commit. PTAL